syntax = "proto3";

package stream_client;

message StartStreamRequest {
  int32 message_count = 1;  // Number of stream messages to receive
}

message StartStreamResponse {
  bool success = 1;
  string message = 2;
}

// Local service
service LocalStreamService {
  rpc StartStream(StartStreamRequest) returns (StartStreamResponse);
}

